### PR TITLE
[ESP32] Fix OTA application READMEs

### PR DESCRIPTION
**Pull request by shubhamdp**
#### Change overview
Fixed the command and some description in ESP32 OTA Requestor and Provider READMEs.

#### Testing
Carried tests with ESP32 OTA requestor and provider with commands in READMEs.

### CODE REVIEW

File: examples/ota-requestor-app/esp32/README.md [hunk 1]
```diff
@@ -31,67 +31,41 @@ idf.py -p <OTARequestorSerialPort> flash monitor
...
+Provider for applying the OTA image. Use the following command from OTA
+requestor prompt
 
 ```
-esp32> ApplyUpdateRequest <OTAProviderIpAddress> <OTAProviderNodeId>
+esp32> ApplyUpdateRequest
 ```
 
-After this step device should reboot and start running hello world example.
+Then reboot the device manually to boot from upgraded OTA image.
```
Comment (dhrishi) [inline examples/ota-requestor-app/esp32/README.md:56]: The README changes look fine. Although, I am not quite convinced with this :-| Why do we have to manually reboot after OTA is done. Is this only for now and will be update/fixed later on?

**Review by dhrishi — APPROVED** [review]

**Review by gjc13 — APPROVED** [review]

File: examples/ota-requestor-app/esp32/README.md (hunk 1 above)
Comment (shubhamdp) [reply examples/ota-requestor-app/esp32/README.md:56]: May be we can reboot automatically after `delayedActionTime`?

**Review by bzbarsky-apple — APPROVED** [review]

**Review by msandstedt — APPROVED** [review]

**Pull request merged**